### PR TITLE
CIP-0119 | Add rationale and context for `doNotList`

### PR DESCRIPTION
**Pull request by Ryun1**
- adding rationale for the somewhat confusing `doNotList` property

### CODE REVIEW

File: CIP-0119/README.md [hunk 1]
```diff
@@ -141,9 +141,10 @@ If the `imageObject` DOES NOT contain a base64 encoded image, the `contentUrl` M
 #### `doNotList`
 - Optional
 - Is a boolean expression that can be given a single value of either `true` or `false`.
-- If not included then the value is assumed to be false
-- A `true` value means that the DRep does not want to show up in tooling that displays DReps. 
-  - e.g. a DRep who does not want to appear in GovTool’s DRep Explorer feature creates metadata with donotlist as true.
+- If not included then the value is assumed to be `false`.
+- A `true` value means that the DRep does **not** want to campaign for delegation via tooling.
```
[the remark this answers is not in the document]
Comment (Ryun1) [reply CIP-0119/README.md:145]: sure!

[the remark this answers is not in the document]
Comment (Ryun1) [reply CIP-0119/README.md:145]: added to rationale via [4a01ee9](https://github.com/cardano-foundation/CIPs/pull/907/commits/4a01ee95049a505ea8f3742975f98acec1797fa9)

**Review by perturbing — APPROVED** [review]

File: CIP-0119/README.md [hunk 2]
```diff
@@ -141,9 +141,10 @@ If the `imageObject` DOES NOT contain a base64 encoded image, the `contentUrl` M
...
+- A `false` value means that the DRep does want to campaign for delegation via tooling and thus be shown via that tooling.
+- e.g. a DRep who does not want to appear in GovTool’s DRep Directory feature creates metadata with `donotlist=true`.
```
Comment (rphair) [inline CIP-0119/README.md:147]:
```suggestion
- e.g. a DRep who does not want to appear in GovTool’s DRep Directory feature creates metadata with `doNotList=true`.
```
... if case is sensitive in this context.

File: CIP-0119/README.md [hunk 3]
```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
 
 Therefore it was decided to allow a provision for people to submit `imageObject`'s with a URL only if a hash was included OR with a base64 encoded image, and allow them to make the decision as to which was most appropriate for their use case.  
 
+#### Rationale for `doNotList`
+
+This field was intended to be used by DReps who wish to identify themselves via rich metadata but are not seeking to campaign for delegations.
```
Comment (rphair) [inline CIP-0119/README.md:194]:
```suggestion
This field was intended for DReps who wish to identify themselves via rich metadata but are not seeking to campaign for delegations.
```
... to be concise (optional)

File: CIP-0119/README.md [hunk 4]
```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
...
+By not be listed via "DRep aggregation/campaign" tools the idea is that these DReps are less likely to attract unwanted delegation from ada holders.
```
Comment (rphair) [inline CIP-0119/README.md:195]:
```suggestion
By not being listed via "DRep aggregation/campaign" tools the idea is that these DReps are less likely to attract unwanted delegation from ada holders.
```

File: CIP-0119/README.md [hunk 5]
```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
...
+These DReps could be organizations want to use their ada to vote, in a transparent way on-chain but do not wish to vote on the behalf of others.
```
Comment (rphair) [inline CIP-0119/README.md:196]:
```suggestion
These DReps could be organizations that want to use their ada to vote in a transparent way on-chain but do not wish to vote on the behalf of others.
```
(grammar)

File: CIP-0119/README.md [hunk 6]
```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
...
+
+It is expected that tooling such as block explorers will list DReps using `doNotList=true`. Whereas tooling built specifically for DRep campaign and delegation should respect the intent of this field.
```
Comment (rphair) [inline CIP-0119/README.md:198]:
```suggestion
It is expected that tooling such as block explorers will list DReps using `doNotList=true`. Tooling built specifically for DRep campaign and delegation should respect the intent of this field. 
```
(grammar)

File: CIP-0119/README.md [hunk 7]
```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
...
 
+#### Rationale for `doNotList`
+
+This field was intended to be used by DReps who wish to identify themselves via rich metadata but are not seeking to campaign for delegations.
+By not be listed via "DRep aggregation/campaign" tools the idea is that these DReps are less likely to attract unwanted delegation from ada holders.
+These DReps could be organizations want to use their ada to vote, in a transparent way on-chain but do not wish to vote on the behalf of others.
+
+It is expected that tooling such as block explorers will list DReps using `doNotList=true`. Whereas tooling built specifically for DRep campaign and delegation should respect the intent of this field. 
+
+This proposal cannot force tooling to respect this desire from DReps. DReps must be aware that any information anchored on-chain can be found via tooling, and may result in delegation.
```
Comment (rphair) [inline CIP-0119/README.md:200]:
```suggestion
This proposal cannot force tooling to respect this desire from DReps. DReps must be aware that any information anchored on-chain can be found via tooling and may result in delegation.
```
(if we are fixing the other commas... this one may be wrong grammar but would be commonly accepted as OK)

**Review by rphair — COMMENTED** [review]
✅ pending...

**Review by rphair — APPROVED** [review]

**Review by perturbing — APPROVED** [review]

**Pull request merged**